ios-swift/CountlyTestApp-iOS-swift/AppDelegate.swift

```diff
@@ -18,14 +18,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate
         config.host = "https://YOUR_COUNTLY_SERVER"
         config.enableDebug = true
 
-//      config.features = [CLYPushNotifications, CLYCrashReporting, CLYAutoViewTracking] //Optional features
+//      config.features = [CLYFeature.pushNotifications, CLYFeature.CLYCrashReporting, CLYFeature.CLYAutoViewTracking] //Optional features
```

Comment:
For Crash reporting feature use this code "CLYFeature.crashReporting" instead of "CLYFeature.CLYCrashReporting"
Remove CLYFeature.CLYAutoViewTracking, as it is deprecated.

Reply:
done